docs/ingestion/data-management.md

```diff
@@ -21,173 +21,9 @@ title: "Data management"
   ~ specific language governing permissions and limitations
   ~ under the License.
   -->
+Within the context of this topic Data management refers to Apache Druid's data maintenance capabilities for existing datasources. There are several options to help you keep your data relevant and to help your Druid cluster remain performant. For example updating, reingesting, adding lookups, reindexing, or deleting data.
 
-
-
-
-## Schema changes
```

Comment:
Why is this section removed?

Reply:
it is 100% duplicated from ../design/segments.md